templates/vue-demo-store/pages/account/order.vue

```diff
@@ -20,8 +56,36 @@ useBreadcrumbs([
   },
 ]);
 
+const limit = ref(route.query.limit ? Number(route.query.limit) : defaultLimit);
+
+const changeLimit = async (limit: Event) => {
+  const select = limit.target as HTMLSelectElement;
+
+  await router.push({
+    query: {
+      ...route.query,
+      limit: select.value,
+      p: defaultPage,
```

Comment:
shouldn't the page be the current page or the same param from router?

Reply:
nope, changing the limit will always bring you back to the default page